site/en/install/pip.md

```diff
@@ -37,6 +44,26 @@ step-by-step instructions.
    python3 -c "import tensorflow as tf; print(tf.config.list_physical_devices('GPU'))"
    ```
 
+* {Windows WSL2}
+
+   Note: TensorFlow is supported on WSL2 on Windows 10 19044 or higher with GPU
+   access is now available. This corresponds to Windows 10 version
+   21H2, the November 2021 update. You can get the latest update from here:
+   [Download Windows 10](https://www.microsoft.com/en-us/software-download/windows10){:.external}.
```

Comment:
Remove hardcoded "en-us" from URL so that the browser will update to whatever language the user prefers: https://www.microsoft.com/software-download/windows10